extensions/amp-autocomplete/validator-amp-autocomplete.protoascii

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-autocomplete
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-autocomplete"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-autocomplete>
+  html_format: AMP
+  tag_name: "AMP-AUTOCOMPLETE"
+  requires_extension: "amp-autocomplete"
+  requires_extension: "amp-form"
+  attrs: {
+    name: "filter"
+    value_casei: "substring"
+    value_casei: "token-prefix"
+    value_casei: "prefix"
+    value_casei: "fuzzy"
+    value_casei: "custom"
+    value_casei: "none"
+    trigger: {
+      if_value_regex: "custom"
+      also_requires_attr: "filter-expr"
+    }
+  }
+  attrs: {
+    name: "filter-expr"
+    requires_extension: "amp-bind"
+  }
+  attrs: { name: "min-characters" }
+  attrs: { name: "max-entries" }
+  attrs: { name: "suggest-first" }
+  attrs: { name: "submit-on-enter" }
+  attr_lists: "extended-amp-global"
+  spec_url: "https://www.ampproject.org/docs/reference/components/amp-autocomplete"
+  amp_layout: {
+    supported_layouts: CONTAINER
+  }
+  mandatory_ancestor: "FORM"
+  reference_points: {
+    tag_spec_name: "amp-autocomplete > input"
+    mandatory: true
+    unique: true
+  }
+  reference_points: {
+    tag_spec_name: "amp-autocomplete JSON"
+    unique: true
+  }
+}
+tags: {  # <amp-autocomplete> > <input>
+  html_format: AMP
+  tag_name: "INPUT"
+  spec_name: "amp-autocomplete > input"
+  mandatory_parent: "AMP-AUTOCOMPLETE"
+  requires_extension: "amp-autocomplete"
+  requires_extension: "amp-form"
+  attrs: {
+    name: "type"
+    mandatory: true
+    value_casei: "text"
+    value_casei: "search"
+  }
+  child_tags: {
+    child_tag_name_oneof: "SCRIPT"
+  }
+}
+tags: {  # amp-autocomplete JSON
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-autocomplete JSON"
+  requires_extension: "amp-autocomplete"
+  mandatory_parent: "AMP-AUTOCOMPLETE"
+  attrs: { 
+    name: "nonce" 
+    disabled_by: "transformed",
+  }
+  attrs: {
+    name: "type"
+    mandatory: true
+    value: "application/json"
+    dispatch_key: NAME_VALUE_PARENT_DISPATCH
+  }
```

Comment:
Please add:

```
  cdata: {
    blacklisted_cdata_regex: {
      regex: "<!--"
      error_message: "html comments"
    }
  }
```

There are some parsing issues in some browsers with comment tag syntax inside of script tags.